### PR TITLE
Update important-upgrade-notes.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -7,6 +7,9 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v5.21.0                                            | Honour key value expiry in KVCompareAndSet, KVCompareAndDelete and KVList. We also improved handling of plugin key value race conditions and deleted keys in     |
```
Comment (amyblais) [inline source/administration/important-upgrade-notes.rst]: Will need more details here.

Comment (lieut-data) [inline source/administration/important-upgrade-notes.rst]: @amyblais, wondering what kind of additional detail to include here. Do we describe the backstory of the bug in question, or just link out to the relevant ticket?

Comment (amyblais) [reply source/administration/important-upgrade-notes.rst]: I was thinking adding a reason why it's a breaking change and which users it affects. I'm not sure though if these are valuable details to add here.

Comment (lieut-data) [reply source/administration/important-upgrade-notes.rst]: Ah, strictly speaking, this is not a breaking change, relative to the intended semantics -- apologies if I miscommunicated on that front. I was thinking we'd just include this in the release notes.

Comment (amyblais) [reply source/administration/important-upgrade-notes.rst]: Ok, sounds good.

**Comment by amyblais** [comment]
@metanerd It seems that I'm not able to merge into the release documentation branch anymore. Is there anything I can do to get permissions?

**Comment by metanerd** [comment]
@amyblais is this still the case now?

**Comment by amyblais** [comment]
@metanerd No, I'm able to merge, thanks!

**Pull request merged**